src/plugins/github/relationalView.js

```diff
@@ -288,6 +288,16 @@ export class RelationalView {
     };
     const raw = N.toRaw(address);
     this._repos.set(raw, entry);
+    this._addCommitHistory(json);
+  }
+
+  _addCommitHistory(json: RepositoryJSON) {
+    if (json.defaultBranchRef != null) {
+      const target = json.defaultBranchRef.target;
+      if (target.__typename === "Commit") {
+        target.history.nodes.forEach((commit) => this._addCommit(commit));
+      }
```

Comment:
If I understand correctly, we believe (at least weakly) that flow can
never reach here. Consequently, we don’t know what the right thing to do
is. Can we throw an error in this case, so that (a) we don’t do anything
unexpected, and (b) we can find out the cause?